web/pandas/community/blog/asv-pandas-grant.md

```diff
@@ -0,0 +1,133 @@
+Title: What we learned on the Pandas Grant
+Date: 2022-03-23
+
+# What we learned on the Pandas Grant
+*By Dorothy Kabarozi B. and Lucy Jiménez*
+
+We want to share our experience working on **Improvements to the**
+**ASV benchmarking framework and diversity efforts** sponsored by
+[NumFOCUS](https://numfocus.org/) to the [Pandas](https://pandas.pydata.org/)
+project.
+
+This grant focused on two aspects: the first one is to improve the
+[asv library](https://asv.readthedocs.io/en/stable/), a tool used by
+benchmarking Python packages and used by Pandas; this project was
+unmaintained, and the codebase was quite old; additionally, it didn't
+adhere to modern standards, had Python 2 compatibility code that could
+be removed, and also the CI could be improved. The second aspect is
+encouraging more underrepresented groups to contribute to Open Source
+projects. This grant was held over 10 weeks, working around 20 hours a
+week. It was developed by Dorothy Kabarozi B. from Uganda and Lucy
+Jiménez from Colombia, under the mentoring of Marc Garcia.
+
+### Why were we part of the grant?
+Even when we come from different backgrounds, Dorothy from systems
+engineering and Lucy from computational chemistry, we have always been
+interested in participating and contributing to open-source software
+projects. For that reason, we have been running the PyLadies meetups in
+our communities (PyLadies Kampala, PyLadies Colombia) and have always
+been on the lookout for any opportunities that led us to be mentored
+contribute.
+
+It all happened through Marc Garcia; he had put out a call on Twitter
+to mentor ladies from diverse backgrounds. Dorothy got to be part of the
+Pandas mentorship group. At the same time, Lucy was co-organizer of the
+SciPy Latam conference, and it is from here she met Marc, who was the
+speaker at that conference, and through this mutual connection, we were
+able to learn about this benchmarks grant.
+
+In brief, by attending conferences, meetups, and social media, you can
+make connections and links that will lead you to these opportunities.
+
+### Learning from the source code
+At the beginning of the grant, we started from the basics. We noticed that
```

Comment:
I'd leave a blank line between the title and the paragraphs. Even if this renders correctly in our website, some markdown flavor fail to render without the right blank lines.